backend/app/core/utils/querying_utils.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any
+
+import bcrypt
+
+
+class QueryingUtils:
+	@staticmethod
+	def hash_password(password: Any) -> Any:
```

Comment:
there is a possibility to type this properly as 
`def hash_password(password: str) -> str:`
 when you also type Auth.password as TextField | str and implicitly convert to str while passing Auth.passoword as argument 
`hash_password(str(self.password))`